protocol/src/main/java/com/amazon/opendistroforelasticsearch/sql/protocol/response/format/FlatResponseFormatter.java

```diff
@@ -26,44 +26,44 @@
 import lombok.Getter;
 import lombok.RequiredArgsConstructor;
 
-@RequiredArgsConstructor
-public class CsvResponseFormatter implements ResponseFormatter<QueryResult> {
-  private static final String INLINE_SEPARATOR = ",";
+//@RequiredArgsConstructor
+public class FlatResponseFormatter implements ResponseFormatter<QueryResult> {
```

Comment:
Could you add some java doc? majorly what is Flat means?

Reply:
Make the base class abstract if possible